cloudformation/scylla.yaml

```diff
@@ -0,0 +1,591 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: >-
+  AWS CloudFormation Scylla Sample Template: This would create a new Scylla Cluster
+
+Parameters:
+  ClusterName:
+    Type: 'String'
+
+  InstanceCount:
+    Description: Number of EC2 instances (must be between 1 and 10).
+    Type: Number
+    Default: 1
+    MinValue: 1
+    MaxValue: 10
+    ConstraintDescription: Must be a number between 1 and 10.
+
+  ScyllaVersion:
+    Type: String
+    Default: ScyllaDB310rc3
+    AllowedValues:
+      - ScyllaDB310rc3
```

Comment:
Why is the version baked into the template?
Will we have a different template for each release?

Reply:
we'll remove the backed ami
since I've check and we can pass params into the urls:
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-console-create-stacks-quick-create-links.html

we'll might still release with every version of the AMI, we'll need to thing about how to match this with the promotion sequence, i.e. talk it over with the releng team